haystack/document_stores/pinecone.py

```diff
@@ -1317,25 +1331,68 @@ def load(cls):
         """
         raise NotImplementedError("load method not supported for PineconeDocumentStore")
 
-    def _meta_for_pinecone(self, meta: Dict[str, Any]) -> Dict[str, Any]:
+    def _meta_for_pinecone(self, meta: Dict[str, Any], parent_key: str = "", labels: bool = False) -> Dict[str, Any]:
         """
         Converts the meta dictionary to a format that can be stored in Pinecone.
         """
```

Comment:
Let's document `parent_key` and `labels`. `labels` especially is not too clear to me.

Reply:
added